### DISCUSSION TITLE
[Feature Request] Add Showcase Section in NextUI Website .

### DISCUSSION THREAD

**Discussion by khashayarghajar** [heroui-inc/heroui#1248] [Feature requests]
### Is your feature request related to a problem? Please describe.

it's important to show off what's possible with NextUI .

### Describe the solution you'd like

1. Go to ' [NextUI](https://nextui.org/) '
2. Click on ' Showcase '

### Describe alternatives you've considered

MUI : `https://mui.com/material-ui/discover-more/showcase/`

### Screenshots or Videos

![Screenshot 2023-02-02 225703](https://user-images.githubusercontent.com/78317297/216430615-4b1dca64-f244-450e-8871-665167684cc4.jpg)

**Comment by munnokd** [comment]
Can I work on this issue?
